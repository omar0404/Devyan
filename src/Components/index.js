import Section from './Section';
import PosterPlaceHolder from './PosterPlaceHolder';
import MovieCard from './MovieCard';
export {MovieCard, Section, PosterPlaceHolder};
