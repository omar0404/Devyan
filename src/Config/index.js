export const POSTER_BASE_URL = 'http://image.tmdb.org/t/p/w500';
