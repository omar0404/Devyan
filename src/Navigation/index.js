import React from 'react';
import {NavigationContainer} from '@react-navigation/native';
import HomeScreens from './HomeScreens';

const RootNavigation = () => (
  <NavigationContainer>
    <HomeScreens />
  </NavigationContainer>
);
export default RootNavigation;
