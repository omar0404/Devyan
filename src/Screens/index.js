import Home from './Home';
import AddMovie from './AddMovie';

export {Home, AddMovie};
