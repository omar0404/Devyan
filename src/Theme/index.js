import Colors from './Colors';

const theme = {
  backgroundColor: Colors.black,
  text: {
    primary: Colors.white,
    secondary: Colors.gray,
    teritary: Colors.lightPink,
  },
  button: {
    default: Colors.lightPink,
    disabled: Colors.gray,
  },
  loadingIndicator: Colors.lightPink,
  posterPlaceHolderBorder: Colors.gray,
  textInputBorder: Colors.gray,
  textInputPlaceHolder: Colors.gray,
  headerShadowColor: Colors.shark,
  sectionLine: Colors.lightPink,
  addMovieIcon: Colors.lightPink,
  headerTintColor: Colors.gray,
};
export default theme;
