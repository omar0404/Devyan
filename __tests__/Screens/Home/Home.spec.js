import React from 'react';
import {shallow} from 'enzyme';
import {Home} from 'MoviesApp/src/Screens';

describe('Home Screen ', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
