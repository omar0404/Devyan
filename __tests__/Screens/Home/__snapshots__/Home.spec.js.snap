// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen  should render correctly 1`] = `ShallowWrapper {}`;
